client/i18n/locales/english/translations.json

```diff
@@ -306,6 +306,7 @@
     "runs-in-vm": "The project runs in a virtual machine, complete the user stories described in there and get all the tests to pass to finish step 1."
   },
   "donate": {
+    "donate-cta":"Donate",
```

Comment:
Missing space here, but I am not that fussed.

Also, we do technically have the word `Donate` already in the translations.